### ISSUE TITLE
A11y_Visual Studio Code Client_Search_ExtensionScoll to Top_Focus Visible: Keyboard focus is not visible on "Scroll to top" button when we are navigating using the tab key

### ISSUE THREAD

**Issue by Ohmry** [Ohmry/todoapp#3]

**Comment by Ohmry** [comment]
TEST
